src/lib/molecules/PosterCard.svelte

```
@@ -1,13 +1,12 @@
+	import Image from "$lib/atoms/DirectusImage.svelte";
+
 	let { name, street, house_number, image } = $props();
 </script>
 
 <li>
 	<a href="/">
-		<img
-			src="https://fdnd-agency.directus.app/assets/{image}?fit=cover&width=400&height=300"
-			alt="Gedenkposter van {name}"
-		/>
+    <Image imageId="{image}" alt="Gedenkposter van {name}" loading="lazy"/>
```

ESLint

[error] Quoted attributes on components and custom elements will be stringified in a future version of Svelte. If this isn't what you want, remove the quotes
https://svelte.dev/e/attribute_quoted(attribute_quoted)

(svelte/valid-compile)